packages/debug/src/browser/style/index.css

```diff
@@ -286,6 +244,10 @@
 .monaco-editor .theia-debug-breakpoint-column.theia-debug-breakpoint-column.debug-top-stack-frame-column {
 	background: url('current-and-breakpoint.svg') center center no-repeat;
 }
+.monaco-editor .theia-debug-top-stack-frame.theia-debug-breakpoint::before,
+.monaco-editor .theia-debug-breakpoint-column.theia-debug-breakpoint-column.debug-top-stack-frame-column::before {
+	content: '' !important;
```

Comment:
@sh7dm mind sharing why are the following rules necessary?

Reply:
It's because of size issue, when using stackframe+breakpoint. We need to hide codicon to let SVG display.

Reply:
> It's because of size issue, when using stackframe+breakpoint. We need to hide codicon to let SVG display.

There must be some styling which is making icon sizes inconsistent, the codicons themselves should all be a standard size.

Reply:
They are consistent, but we need breakpoint's dot to be inside an arrow.

Reply:
> They are consistent, but we need breakpoint's dot to be inside an arrow.

Do you mind sharing a screenshot? Possibly the same icon in VS Code?

Reply:
Hard to screenshot it now, but I'll explain. That's stackframe icon overlapped by breakpoint one. Yellow arrow and red circle over it. Circle is too big, bigger than it should.
> Possibly the same icon in VS Code?

There is, maybe we can try to find it.

Reply:
I found that! They use `debug-stackframe-dot` instead of big breakpoint!